cypress/e2e/checks.test.ts

```diff
@@ -55,6 +55,84 @@ describe('Checks', () => {
     cy.getByTestID('save-cell--button').should('be.enabled')
   })
 
+  it('can validate a deadman check', () => {
+    //create deadman check
```

Comment:
shouldn't the linter be doing something with these comments?

Reply:
once i merge the rule in, yes